src/prx/rinex_nav/evaluate.py

```diff
@@ -657,6 +662,12 @@ def evaluate_orbit(sub_df):
     if "signal" in per_signal_query.columns:
         columns_to_keep = ["signal", "sat_code_bias_m"] + columns_to_keep
     columns_to_keep.append("frequency_slot")
+    computed_columns_to_keep = [
+        col for col in columns_to_keep if col not in query_columns
+    ]
+    per_signal_query.loc[
```

Comment:
If we don't do this, we output sat states computed from the most recent ephemeris we have, even if it is 10 days old, see added unit test https://github.com/jtec/prx/blob/f14f095861a373a16ce4760dc7a42847d1005bbe/src/prx/rinex_nav/test/test_evaluate.py#L87